lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -232,6 +168,31 @@ defmodule MembraneLiveWeb.EventChannel do
     {:noreply, socket}
   end
 
+  def handle_in(
+        "presenting_request",
+        %{"email" => email},
+        socket
+      ) do
+    {:ok, _ref} = Presence.update(socket, email, &%{&1 | request_presenting: true})
+
+    "event:" <> id = socket.topic
```

Comment:
it's just a suggestion would it be better to create id in function pattern match?